### DISCUSSION TITLE
Still worth plotting? vs. buy XCH directly?

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#5736] [Ideas]
With the netspace today and a growth speed of 10% a day, every 100TiB of plots would only be able to bring about ~1XCH in the entire life, yet it costs about $2k-$3k to get the drive new, not including all the plotting cost, HDD wear, and electricity bills.  Is it really worth to continue this journey? Unless you can lower down the entry cost for the HDD and the plotting cost, it looks like a guaranteed negative return scenario based on all the hardware cost alone.  Wonder if it makes sense to convert all the expenditures directly to XCH instead? You would get 1XCH for each $830 instantly now, yet comparing with say $3000 on the rig and wait for a year???

**Comment by katmai** [comment]
if you regard chia and cryptocurrency as pure speculation and you're not in it for the tech, trying to see what opportunities will it open for us, if it will be adopted, if it solves any world problems, trying to support the network succeed, understanding what cryptocurrency and community is all about, learning new technologies, then your points are pretty valid and it's fairly pointless to start farming/plotting and the future is pretty bleak.

**Reply by pluto2021** [reply]
Especially with the delay on the official pool, it is hard on people's wallet to keep growing the netspace.  This will soon turn into rich people's game.  With small amount of HDD space each small guy own today, it may take years to see the luck dropping from sky, yet you have to keep the HDD running 7/24. Taking into account the HDD wear over time, and the electricity bill over the years, it is hard to justify the expenditures. We certainly hope that the the Chia blockchain concept to be a successful one in the long run. However, you cannot just expect that people would just keep throwing their hard earned money into it with negative ROI.  Any successful business model would take  the investor's interest into account seriously. In some sense, we are all investors. If the Chia blockchain technology cannot bring positive ROI on every plot made, it may not be sustainable. Due to the life expectance of an HDD, each plot also have its lifespan. 

Once the official pool would open soon, you would have to replot???!!! HPool will certainly not replot! So the new pools will still have hard time competing with HPool for example.  As a center of HDD manufacturing in Asia (e.g. lowest HDD cost from factory direct, for big guys who can buy in bulk and negotiate pricing, not the small guys), I am not sure if the new pools would be able to keep up and still bring positive ROI for each new plot made. The XCH pricing would have to exceed that of BTC not very far into the future to keep a positive ROI. Guess who will still be the biggest winners, the whales.

**Reply by thewhitegrizzli** [reply]
Very on point facts. Agreed 100%.

**Reply by room101-dev** [reply]
Well to be honest, its no different that GPU mining, my gtx-1060 cards die quite often, 90% I can fix them, these days you can't replace them, as there none to buy

I make $40/day on each HPOOL mining rig, average about 400 plots per rig, In terms of cost its exact same results I get form mining ETH. Mostly I kill NVME drives to make the plots.

I don't need anything chia-dev has to offer to make plots, and I don't actually even need to run the blockchain ( which doesn't work) I think CHIA will fork into a dozen different verions, with HPOOL version, but #1 that I will  use. They pay fair and well.
That's another thing about the chia-pool(non-existent) that plan on charges fee's 10 different ways, which is why they hate Hpool.
Also HPOOL will never ask you to delete your old plots, where Cohen already says you must adopt his new "portable-plot" format, who knows why they didn't get it right the first time? My guess is this is just to kill hpool.
Majority is already on hpool, especially in asia, everybody uses it, in fact they don't even run on of the buggy chia software, they have their own plotter & miner.

I'll repeat that HPOOL has an option to run their plotter, and their mining-pool client, you never even have to go to github to mine&plot chia using HPOOL's software. I myself use chia-pos to plot and hpool-miner, but because for plotting I want access to the source.

...

Hard-disks don't die, except for infant mortality or you drop them, farming or 'mining' is very low usage, mostly their just reading the header, the cpu cycles is quite high, but the disk utilization is quite low when the HPOOL miner is running.

The only disk abuse of chia, is the making of plots, where you must have NVME's the -t temp-dir kills the controller after about 100 plots, well that's what I'm seeing EVO 980 pro 1tb, I run three on each plotter

**Reply by pluto2021** [reply]
Hard-disks do "die", or starts to go bad (bad sectors, bytes, bits, etc.) that can corrupt the file table, the files, or any other data stored within. Make sure that you examine some old HDDs using some software tools. Have a 10 yr old Seagate SATA enterprise grade HDD with a lot of files on it and CrystalDiskInfo showed some issues, and I know some files on it are corrupted. Based on the Seagate specification, the MTTF should be 1.4Mhrs. My drive only had <90khrs life. It did not die yet, but becomes problematic. While this HDD was mostly off and only occasionally on over the past 10 yrs. For Chia plots, the HDDs would have to be spinning 7/24. It is a lot of wear and heat. If you do not actively cool you HDDs, they could burn out prematurely very quickly.  That's one reason for the fan arrays aligned with the HDDs on many servers.

BTW, electricity bill do mount up linearly with time and can become quite significant, assuming you would not scale up at a same rate with the netspace size.

**Comment by alexs20** [comment]
Don't and Don't
As you correctly said with current Netscape growth the plotting is pointless and will never bring ROI.
On another side, looking back on how the network was launched I can see it as a big failure dived by greediness. 
They rushed to launch it without building proper pooling ecosystem driven by desire to see how rich they became with their 21M of pre-mined coins, ignoring the fact that decentralized crypto-currency is always a symbiose with army of small farmers who usually participates only because of small revenue. We are in capitalistic world, nobody will farm for nothing and only few short-sighted percents will be inspired by fiery speeches about solving future problems and building better currency. 
Today HPool consumes more and more small unhappy farmers, and I see that even not small farmers switching to HPool but relatively big. Saying that, decentralization doesn’t grow anymore, people loosing the trust, the future pool implementation will not be able to compete with HPool and all these facts affects chia price. Therefore investing in chia IMHO hopeless.

**Reply by pluto2021** [reply]
Another hypothesis is that supercomputer clusters might have been used in the Chia plotting that no one can compete! Those supercomputers are usually government build, might usually have very low utilizations, and huge storage spaces already. They could probably easily complete a plot in say every a few seconds! The decentralization scheme might have already been broken. There is first the centralization on the time (early birds got significantly more rewards), then the centralization of computation power and net space. So even if you already own some HDD spaces, due to the hardware lifespan, it might not even worth the pain. The main net was launched very immaturely and small farmers get slaughtered.

**Reply by thewhitegrizzli** [reply]
Clearly already broken since it never existed. "The main net was launched very immaturely and small farmers get slaughtered."=> YES.

**Reply by room101-dev** [reply]
> Don't and Don't
> As you correctly said with current Netscape growth the plotting is pointless and will never bring ROI.
> On another side, looking back on how the network was launched I can see it as a big failure dived by greediness.
> They rushed to launch it without building proper pooling ecosystem driven by desire to see how rich they became with their 21M of pre-mined coins, ignoring the fact that decentralized crypto-currency is always a symbiose with army of small farmers who usually participates only because of small revenue. We are in capitalistic world, nobody will farm for nothing and only few short-sighted percents will be inspired by fiery speeches about solving future problems and building better currency.
> Today HPool consumes more and more small unhappy farmers, and I see that even not small farmers switching to HPool but relatively big. Saying that, decentralization doesn’t grow anymore, people loosing the trust, the future pool implementation will not be able to compete with HPool and all these facts affects chia price. Therefore investing in chia IMHO hopeless.

If your not already on HPOOL months ago you are an idiot. The dev-team tells you to wait and get zero

I will tell you this, I bet you all that dev-team has been on HPOOL all the time, and why not? What good for the goose, is not good for the gander in chia-cult land. Everybody here knows this is a scam, so why tell people not to pool-mine?

Its the first thing I did when I started plotting months ago, was jump on HPOOL, I knew that solo-mining would never pay, today I have lots of XCH from mining my plots on hpool.

I agree with everyone here, main-net never existed, it will never sync, because 100% of all the code is shit, the entire project is a scam, all marketing and no development. Dozens of paid influencers on youtube all reading from the same script for months. All paid in XCH, ask CALEB, How many XCh you got?? He will not tell you, but he says not to mine on hpool, that tells you right there that we was 'gifted' XCH by dev, for promoting, e.g. talking BS on his channel, most the BS is anti-hpool, after wonder why? I think its because the porn king ( cohen ) of bittorrent, doesn't want to lose control of the mining, as then you can't fill the plots with encrypted CP.
CHINA now controls CHIA forever, its not coming back to Cohen or Hoffman.

**Reply by pluto2021** [reply]
Developing a pool after the official pool protocol always need to carry an incentive to run as a business. If there is no incentives to do so, there would never be any sustainable pools. Caleb most likely got on the wagon early, so there might not be anything to blame there actually, when an ancient computer can make some xch. It is natural to tell people to wait for the official pool, otherwise the cost of developing a new pool would never get recovered and paid back. If telling everyone to jump on hpool, who would wait for the official pool? But the cold facts is there, every day of waiting, your invested plots would value less, in an exponential manner. So if you still are not making XCH solo, your plots would statistically become worthless very shortly. By the time the official pool is out, you may need 10-100x the amount of plots you have now to make a same profit as today! Yet, what is the ROI on those 10-100x amount of plots you have to make? The only way this could ever work out is that the XCH price would sky rocket. With the current climate of crypto meltdown, unless you are a true believer who is willing to hold the bags, waiting for that day to happen, it is hard to justify for even a single plot under the official pool.  Ironically, if you really believe that xch would skyrocket soon, why not just buy some xch as well if you could now? At least, you got some xch instantly? I bet the founders are working hard to solve this problem. Hope that they are with the majority of the farmers and manage to find a resolution timely.  

One potential reason on the delay of the official pool might be that it has been asked too much of a functionality that has led to the complexity of the pool protocol architecture making it difficult to develop. There could be a lot of bugs once that is out.  Why the founders would not just roll out a simple pool protocol without all the other bells and whistles first before adding complexity on the top?

**Comment by thewhitegrizzli** [comment]
I think this project had huge potential, but unfortunately will go down the drain of the shitcoins river for these reasons above, and others. However, I also think it really paved the way for future Proof-of-Space/Time coins...so...I'll just let the plots I have now to farm their way and see what happens, but as of 22.05.2021 I will stop all plotting activities on my servers and also I cancelled hundreds of terabytes I had ordered and will return a bunch of disks for refund before the refund window closes.... RIP XCH.

**Reply by room101-dev** [reply]
This project has no use, if it does please tell us. I only jumped in because I can't expand my GPU mining hobby because GPU's are no longer available. When I quit chia, I'll probably do filecoin, or look closely at another 'POC' coin, why not, its good to diversify. The problem at least in file-coin they use the storage, in CHIA all that EB is wasted, the company who develops the miner to tap all that storage space for encyrpted data has the keys to the kingdom. Think about this COHEN got rich with bittorrent by stealing music and movies, so he will think nothing of stealing our drives and fill the .plot files with his porn, or CP for his clients.

The chinese however want nothing of this, which is why HPOOL even exists, its to make sure the west doesn't control what gets stored on the .plot files in the future, all that matters is the name of  the file and the header, the reason 108gb of space that can be 'sold' and/or rented just like file-coin.

**Comment by room101-dev** [comment]
> With the netspace today and a growth speed of 10% a day, every 100TiB of plots would only be able to bring about ~1XCH in the entire life, yet it costs about $2k-$3k to get the drive new, not including all the plotting cost, HDD wear, and electricity bills. Is it really worth to continue this journey? Unless you can lower down the entry cost for the HDD and the plotting cost, it looks like a guaranteed negative return scenario based on all the hardware cost alone. Wonder if it makes sense to convert all the expenditures directly to XCH instead? You would get 1XCH for each $830 instantly now, yet comparing with say $3000 on the rig and wait for a year???

You can mine on HPOOL with your plots today, and yesteday, there is no reason not to be there. Why carry on like there is no pool? HPOOL works just fine and pays out.

1 plot pay 7 cents/day, if you go to the chia-calc, on the bottom the guy is offering plots for sale, click and you will see $15/plot

It takes 214 days for one plot to pay for itself, the problem of course is that EB(space) doubles every 9 days, so your profit goes down 1/2 every 9 days. So the compounding loss means it would take years to payback the cost of one plot.

**Reply by padik11** [reply]
I am trying to join from hpool but the process is not well described. Can anyone help who has already joined there?

**Comment by GeorgePapageorgakis** [comment]
Sort answer, **not anymore** unless you can withstand the forecoming bear market where each chia will probably cost less than 1K $ and we still dont know for how long it will last.

Problem is that chinese farmers are all switching from gpu farming, to hdd farming so... the net will continue to grow at that pace for the comming months :(

**Reply by padik11** [reply]
Are you saying that growing may not make sense?
